joomla/joomla-cms
# [4.3] Missing php fileinfo

Pull Request for Issue #39711 .

### Summary of Changes
When the php fileinfo module is not installed then media manager will give the following error

> Error Property 'mime_type' of argument 'item' of event onFetchMediaItems has a wrong item. Valid: string

This PR adds a check for the php fileinfo module and displays a more helpful error message

> The Fileinfo PHP module is not installed. You should contact the server administrator to install it.

### Testing Instructions
To display the error message
Either disable the fileinfo module in your php.ini and open the media manager
or edit the check to reverse it so it shows when the module is present by removing the ! in (!\extension_loaded('fileinfo')) {

### Link to documentations
Please select:



There may be changes required in the docblock - no idea
